redis.go

```
@@ -186,11 +178,9 @@ func (c *RedisDB) changeUserPassword(ctx context.Context, username, password str
 	var redisErr resp3.SimpleError
 	err = db.Do(ctx, radix.Cmd(&mn, "ACL", "GETUSER", username))
 	if errors.As(err, &redisErr) {
-		fmt.Printf("redis error returned: %s", redisErr.S)
+		fmt.Errorf("redis error returned: %s", redisErr.S)
```

We'll want to return this error.